common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -795,6 +797,61 @@ def test_courseware_nav(self):
         self.course_nav.go_to_section('Test Section 2', 'Test Subsection 3')
         self.assertTrue(self.course_nav.is_on_section('Test Section 2', 'Test Subsection 3'))
 
+    # def test_course_home(self):
```

Comment:
This commit was a sneak preview of changes to get review on the use of `course_home.outline` before a larger refactor.  I will do some additional clean-up to make this clear.